doc/source/installation.rst

```diff
@@ -1,140 +1,209 @@
+OpenMP support.
+
+**MDAnalysisTests** is optional and it is a separate :ref:`test suite <mdanalysistests>` used 
+for verifying MDAnalysis installations and running User Guide examples. It is not required for 
+daily use, but some tutorials rely on it. The package is approximately 90 MB and is not updated frequently. 
+If you plan to use tutorial examples or contribute to MDAnalysis, installing it is recommended.
+
+MDAnalysis requires Python 3.8 or later. Ensure you have an appropriate Python version installed before proceeding.
 
 .. note::
+    MDAnalysis supports **Linux**, **macOS**, and **Windows** (Python 3.8+). 
 
-    If you are installing on Windows, you must have
-    Microsoft Visual C++ 14.0 installed. If your installation
-    fails with the error message:
+    - If you encounter errors on **Windows** related to **Microsoft Visual C++ 14.0**, install the required **Build Tools for Visual Studio** from: `Microsoft Visual Studio Downloads`_. 
+    - If you encounter any other issues following these instructions, seek help on `GitHub Discussions (Installation)`_.    
 
-        error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio": https://visualstudio.microsoft.com/downloads/
+.. warning::
+    :ref:`mamba-installation` and :ref:`pip-installation` installations **do not include** external programs such as `HOLE`_.
 
-    Try installing Build Tools for Visual Studio from
-    https://visualstudio.microsoft.com/downloads/ (scroll
-    down to the Tools for Visual Studio section).
+.. _mamba-installation:
 
+mamba/conda 
+===========
 
-If you encounter any issues following these instructions, please
-ask for help on `GitHub Discussions (Installation)`_.
+For most users, `mamba`_ is the **recommended** way to install MDAnalysis. It is a faster drop-in 
+replacement for `conda`_ and efficiently handles dependencies. We also recommend creating a new 
+environment for MDAnalysis to ensure a clean and isolated installation.
 
-.. _`GitHub Discussions (Installation)`: https://github.com/MDAnalysis/mdanalysis/discussions/categories/installation
+MDAnalysis supports two common ways to install with `mamba`_:
 
-conda
-=====
+**1. Recommended: Use Mambaforge**
 
-If you use ``conda`` to manage your Python environment, we highly recommend creating a new environment for MDAnalysis.
-This will ensure that you have a clean installation of MDAnalysis and its dependencies, and will not interfere with other packages you may have installed.
-We further recommend that you install and use ``mamba``, a faster drop-in replacement for ``conda``.
+`Mambaforge`_ is a minimal `conda`_ distribution that includes `mamba`_ by default, uses the `conda-forge`_ channel 
+(preferred for MDAnalysis) and avoids clutter from `Anaconda`_ base packages.
+
+To install MDAnalysis with `Mambaforge`_, first download the `mambaforge installer`_ and then run: 
```

Comment:
I think here needs to be something like "follow the mambaforge installation instructions and then you should be able to run the following commands to install MDAnalysis:"